<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HanZi</class>
 <widget class="QWidget" name="HanZi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>694</width>
        <height>514</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLeft">
       <property name="topMargin">
        <number>20</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labPinYin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>258</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>258</width>
         <height>60</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#aaaaff;&quot;&gt;拼音&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="labGif">
       <property name="minimumSize">
        <size>
         <width>258</width>
         <height>258</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>258</width>
         <height>258</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#aaaaff;&quot;&gt;笔画&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#aaaaff;&quot;&gt;演示&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>126</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>126</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>首页</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnStart">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>126</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>126</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开始滚动</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>126</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>126</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>126</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>126</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>258</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>258</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>输入汉字</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QCheckBox" name="checkBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑 Light</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开启/关闭语音</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>HanZi</receiver>
   <slot>addText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>801</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>HanZi</receiver>
   <slot>delText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>916</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>714</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addText()</slot>
  <slot>delText()</slot>
 </slots>
</ui>
